rest-server/src/main/resources/nubesgen/terraform/modules/app-service/main.tf.mustache

```diff
@@ -254,6 +254,44 @@ resource "azurerm_linux_web_app" "application" {
     "SPRING_DATA_MONGODB_URI"      = var.azure_cosmosdb_mongodb_uri
   {{/addonCosmosdbMongodb}}
 {{/runtimeSpring}}
+{{#runtimeMicronaut}}
+    "MICRONAUT_ENVIRONMENTS"       = "prod,azure"
+  {{#runtimeDocker}}
+    "MICRONAUT_SERVER_PORT"        = 8080
+  {{/runtimeDocker}}
+  {{#runtimeJava}}
+    "MICRONAUT_SERVER_PORT"        = 80
```

Comment:
is port 80 intentional?

Reply:
Yes.  The webapp serves on port 80 when run as a java app